packages/x-components/src/__tests__/utils.ts

```diff
@@ -194,16 +193,16 @@ function mergeStates<State extends Dictionary>(
  */
 export function installNewXPlugin(
```

Comment:
I'm refactoring this in another PR which I will open soon. We could discuss it then and there but I would like to keep the function because we would need to use quite a lot in the tests, so we can save several lines of code. 

My proposal will be:

![image](https://github.com/user-attachments/assets/07f856b9-cc7f-4f0e-858b-086f50f8702b)

And the default use would be to use just a line:

`global: { plugins: [installNewXPlugin()] },`

Reply:
PR adopting these changes: #1586